### ISSUE TITLE
Port to Android/iOS

### ISSUE THREAD

**Issue by UnAfraid** [winauth/winauth#399]
If you port this application on Android or iOS i will not use original Google Authenticator, this one is awesome!
Do u have any plans on porting it?
Keep up the good work!

**Comment by beerisgood** [comment]
For Android it would be nice if you include F-Droid support!

**Comment by winauth** [comment]
While a nice idea, there are already a lot of Android versions of similar apps, Google Authenticator, Authenticator+, Authy, etc. I'm not sure what WinAuth would be adding or how it would be unique from the others.

If you have a specific need that these ones aren't addressing, then maybe I could look into it more, but really WinAuth is intended as a convenient version for Windows or for those without access to a phone.

**Comment by UnAfraid** [comment]
I really like how win auth is made, the ability to put icons on each is really useful the password protection import/export of the data is really awesome.

On android everytime i flash some another room / factory reset i am loosing all the authenticator's data if i haven't backup it on winauth i would be screwed up real bad if i don't remove it first.

**Comment by flasheater** [comment]
@winauth I think you a right, Android has a lot of fine options in that regard. @UnAfraid Check out https://fedorahosted.org/freeotp/. 
The notable exception here is support for games (battle.net, steam,etc). Those require their own proprietary clients. Maybe you could document how you reverse engineered for example Battle.net support?

**Comment by robinhugy** [comment]
yeah, please port to android (maybe iphone) its a good software, i can use only this for all dont need 4-5 or more s**t app

**Comment by wildsprite** [comment]
I know I'm bumping an old thread but please please please make android and ios ports of this, it is vastly superior to the others, the only thing missing from it is the Square-Enix Security Token but I doubt that one would be easy to do unless you wanted to try to emulate the physical token but I'm not even sure

**Comment by ShadowOfDeimos** [comment]
@winauth I'm also bumping this thread.  You're right, there are a lot of other authenticators to choose from on android.  However, the other authenticators do not support battle.net and steam.  This is the first authenticator  which includes every authenticator I need.  I would love to see this project grow because I love the windows app's simplistic to-the-point UI.  I also love that this one is open source.  Perhaps a port using xamarin would make cross-platform support easier.

**Comment by Bluscream** [comment]
Yeah i have to admin that most people only have the huge steam app for the auth and confirmation stuff on their phone. If there was one small app to compensate this, i would get rid of useless steam for android

**Comment by frozenspider** [comment]
Bumping this up as well. Another huge killer feature of having WinAuth for Android/iOS would be an ability to sync authenticators between Windows and smartphone.
